code/style.css

```diff
@@ -2,6 +2,327 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+
+
+/* for smooth scrolling when pressing the animation */
+html{
+    scroll-behavior: smooth;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    max-width: 100vw;
+}
+
+
+/* Hero image and text */
+.hero {
+    background: url('https://images.unsplash.com/photo-1621605815971-fbc98d665033?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80') center;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    max-width: 100vw;
+    height: 100%;
+    overflow-x: hidden;
+}
+
+.bigtext {
+    font-size: 10vw;
+    padding-bottom: 1rem;
+    color: #EFEEED;
+    text-decoration: underline;
+}
+
+/* The div around the animation */
+.scroll-down {
+    height: 50px;
+    width: 30px;
+    border: 2px solid orange;
+    position: absolute;
+    bottom: 20px;
+    border-radius: 50px;
+    cursor: pointer;
+    display: flex;
+    align-content: center;
+}
+/* I had problems centering it without destroying the animation it is 1% off, live with it please :> */
+.halfling {
+    left: 49%;
+    margin: auto;
+}
+
+/* the animation and keyframes */
+.scroll-down::before,
+.scroll-down::after {
+    content: "";
+    position: absolute;
+    top: 20%;
+    left: 50%;
+    height: 10px;
+    width: 10px;
+    transform: translate(-50%, -100%) rotate(45deg);
+    border: 2px solid orange;
+    border-top: transparent;
+    border-left: transparent;
+    animation: scroll-down 1s ease-in-out infinite;
+}
+
+.scroll-down::before {
+    top: 30%;
+    animation-delay: 0.3s;
+}
+
+@keyframes scroll-down {
+    0% {
+        opacity: 0;
+    }
+    30% {
+        opacity: 1;
+    }
+    60% {
+        opacity: 1;
+    }
+    100% {
+        top: 90%;
+        opacity: 0;
+    }
+}
+
+
+/* second section with the forms */
+.subscribe {
+    background-color: orange;
+    display: flex;
+    justify-content: center;
+    margin: 0 auto;
+    position: relative;
+}
+
+.bg {
+    background-image: url(https://images.unsplash.com/photo-1621607512026-ba3e0857c6f4?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=634&q=80);
+    background-size: cover;
+    width: 50vw;
+    height: 100vh;
+    margin-top: 1rem;
+}
+
+.forms {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    max-width: 100vw;
+    position: absolute;
+    position: relative;
```

Comment:
One thing I noticed was that in the different mobile views / and when the window gets a little big squeezed vertically, your sign up-form element and scroll-down-button "bleeds" out into/on top of the FAQ-section and ontop of each other aswell. I tried to remove the positions: absolute and position: relative in the inspection-tool (is there a reason that you have both of them at the same time?)  and it seems like it doesn't bleed out if I do that, instead it stays separated above the FAQ. but it would need some padding or margin between the subscribe-button and the scroll-down-button. because now that are very tight together. I hope you followed what I meant, can be a bit tricky to write these things. let me know if you need clarification! 😊